bundles/org.eclipse.mita.program/src/org/eclipse/mita/program/inferrer/StaticValueInferrer.xtend

```diff
@@ -113,4 +113,4 @@ class StaticValueInferrer {
 		inferenceBlockerAcceptor.apply(expression);
 		return null;
 	}
-}
\ No newline at end of file
+}
```

Comment:
Nothing changed in this file except whitespace got worse. please revert this file to master.

Reply:
reverted